platforms.md

```diff
@@ -79,7 +79,7 @@ create images from the image file.
  - The DO platform wraps [godo](https://github.com/digitalocean/godo).
  - By default SSH keys will be passed via both the DO metadata AND the userdata.
  - UserData is passed to the instances via the DO metadata service.
- - DigitalOcean has no method for uploading custom images, as a result the `ore do create-image` command does a [~~terrifying~~ special workaround](https://github.com/flatcar-linux/mantle/blob/master/cmd/ore/do/create-image.go#L117-L173) which specifies custom userdata that does the following (after which the machine is snapshotted):
+ - DigitalOcean has no method for uploading custom images, as a result the `ore do create-image` command does a [~~terrifying~~ special workaround](https://github.com/flatcar/mantle/blob/master/cmd/ore/do/create-image.go#L117-L173) which specifies custom userdata that does the following (after which the machine is snapshotted):
    - configure networking in the initramfs
```

Comment:
This part is not true anymore, as far as I remember

Reply:
Dunno, these docs would need a review anyway, but there is a lot of more important things to work on…